src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_NUMBER = 100;
+    private final Random random;
+
+    public Lottery(Random random) {
```

Comment:
do you really need this 'random' parameter here? I mean, you are assigning your class 'random' field the 'new Random()' and constructor parameter isn't used at all . So, I'd recommend removing this constructor parameter